lib/new_relic/agent/instrumentation/grpc/client/chain.rb

```diff
@@ -29,7 +29,7 @@ def bidi_streamer_with_new_relic_trace(method, requests, marshal, unmarshal,
               metadata: {},
               &blk)
 
-              issue_request_with_tracing(method, requests, marshal, unmarshal,
+              issue_request_with_tracing(:bidi_streamer, method, requests, marshal, unmarshal,
```

Comment:
- Have each instrumented client request sending method report its name to
  `issue_request_with_tracing`, and then report that name was `grpc_type`